apis/nucleus/src/components/listbox/components/useTempKeyboard.js

```diff
@@ -53,12 +53,12 @@ export default function useTempKeyboard({ containerRef, enabled }) {
       const fieldElement = c?.querySelector('.value.selector, .value, .ActionsToolbar-item button');
 
       const elementToFocus = searchField || lastSelectedRow || fieldElement;
-      elementToFocus?.setAttribute('tabIndex', 0);
+      elementToFocus?.setAttribute('tabIndex', -1);
```

Comment:
Maybe we can remove these `elementToFocus?.setAttribute('tabIndex', -1);` completely if we just make sure not to set them to `0` in other places (if that's the case…).